docs/updating/versioning-policy.md

```diff
@@ -3,75 +3,100 @@ menu-title: Versioning policy
 meta-title: Versioning and release process | CKEditor 5 Documentation
 meta-description: How CKEditor 5 is versioned and released - aligned versions across packages, release channels (stable, nightly, alpha/RC), and guidance on staying up to date.
 category: updating
-order: 50
+order: 40
```

Comment:
Since this is a single guide, and it's followed by all the changelogs now, I reordered them. Changelogs are last now.